flows/actions/forward_ivr.go

```diff
@@ -0,0 +1,60 @@
+package actions
+
+import (
+	"github.com/nyaruka/gocommon/urns"
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/events"
+)
+
+func init() {
+	registerType(TypeForwardIVR, func() flows.Action { return &ForwardIVRAction{} })
+}
+
+// TypeForwardIVR is the type for the forward IVR action
+const TypeForwardIVR string = "forward_ivr"
+
+// ForwardIVRAction can be used to forward an ongoing IVR call to a different phone number.
+//
+// An [event:ivr_forwarded] event will be created with the evaluated phone number.
+//
+//   {
+//     "uuid": "8eebd020-1af5-431c-b943-aa670fc74da9",
+//     "type": "forward_ivr",
+//     "phone": "+12065551212"
+//   }
+//
+// @action forward_ivr
+type ForwardIVRAction struct {
+	baseAction
+	voiceAction
+
+	Phone string `json:"phone" validate:"required" engine:"evaluated"`
```

Comment:
@rowanseymour before I go to much farther on this, any strong feelings on this being represented as a URN vs a text named `phone`? I *think* phone numbers are the only valid value here and it should be an expression so `phone` felt more right here than say `urn`, but on the other hand for the evaluated event, `urn` definitely feels more right. Opinions?

Reply:
Just checking you know https://github.com/nyaruka/goflow/pull/972 exists?

Reply:
Agree event should be URN, and despite what I have in that PR, agree in shouldn't be a validated URN field on the action. Phone.. maybe tying our hands a little.. whatsapp voice forwards? Could be scheme/path like add_contact_urn action

```
Scheme string `json:"scheme" validate:"urnscheme"`
Path   string `json:"path" validate:"required" engine:"evaluated"`
```

Reply:
Oh no, didn't know about #972.. doh. Will take that over.

I suppose I'm fine with Scheme/Path though wouldn't want to expose that in the UI.

Reply:
Yeah just phone in the editor for now

Reply:
Although, thinking through this further having this be settable doesn't seem valid. I mean, you can't forward a physical phone call to whatsapp or vice versa (and it is hard to imagine us ever somehow supporting that), so why even open up the possibility of that being out of sync? Maybe we should just derive the scheme from the current contact scheme?

Reply:
True, then just `path`?

Reply:
I'm ok with `path` but would still vote for `phone` just because it feels clearer. Your choice though.